scripts/docker/SERVER.md

```diff
@@ -0,0 +1,20 @@
+## Server side
+
+The state of this document and the associated script is beta, please use with caution.
+
+To start a borg server:
+ - generate an ssh-key
+   - `ssh-keygen -f ./id_rsa -N '' -t rsa`
```

Comment:
It's 2021, not sure whether recommending RSA keys is still what we need. ED25519 could be used just as that, is shorter and at least as secure…